core/src/main/java/org/apache/iceberg/jdbc/JdbcUtil.java

```diff
@@ -135,7 +135,7 @@ final class JdbcUtil {
           + CATALOG_NAME
           + " = ? AND "
           + TABLE_NAMESPACE
-          + " LIKE ? LIMIT 1";
+          + " LIKE ? ";
```

Comment:
I think this should be an exact search ("="), what's the rationale for "LIKE"?

Reply:
Hey @amogh-jahagirdar i think it's due the fact that JDBC catalog support nested namespace, so assuming we have a valid namespace like foo.bar the namespace exist shall return true when foo alone was queried. @ismailsimsek please correct me if i am wrong.

Reply:
@amogh-jahagirdar @dramaticlly correct its for nested namespaces

Reply:
Why does this remove the LIMIT?

Reply:
rolled this change back